include/NovelRT/Maths/Utilities.h

```diff
@@ -70,6 +93,11 @@ namespace NovelRT::Maths
             return Log2DeBruijn()[(value * 0x07C4ACDCu) >> 27];
         }
 
+        /**
+         * @brief Count the preceeding zeroes of a given 32 bits unsigned integer
+         * @param value Then given integer to count the preceeding zeroes
+         * @return Number of zeroes proceeding the given value
+         */
```

Comment:
I'm not entirely sure I understand what you were trying to convey here, so I've left my changes at punctuation and grammar.

```suggestion
        /**
         * @brief Counts the preceeding zeroes of a given uint32_t.
         * @param value The given integer to count the preceeding zeroes on.
         * @return The number of zeroes preceeding the given value.
         */
```